Library/Homebrew/global.rb

```diff
@@ -30,7 +30,7 @@
 HOMEBREW_USER_AGENT_CURL = ENV["HOMEBREW_USER_AGENT_CURL"]
 HOMEBREW_USER_AGENT_RUBY = "#{ENV["HOMEBREW_USER_AGENT"]} ruby/#{RUBY_VERSION}-p#{RUBY_PATCHLEVEL}"
```

Comment:
Perhaps it should be, in `brew pull`. My new code isn't setting the user agent anywhere, I think.